scripts/globals/caskets.lua

```diff
@@ -185,7 +185,7 @@ local function dropChance(player)
     --end
 
     if player:hasStatusEffect(xi.effect.PROWESS_CASKET_RATE) then
-        prowessCasketsPower = casketProwessEffect:getPower()
+        prowessCasketsPower = casketProwessEffect:getPower() / 100
```

Comment:
While you're here and you understand the code, can you change the `local rand = math.random()` below this to be `math.random(1, 100)` (like we're trying to use in the rest of the code?

Would that remove the need for this /100 divisor?

Reply:
I can change/update it. May need to make some adjustments.

xi.settings.main.CASKET_DROP_RATE = 0.10 Default Setting.

kupowersMMBPower and prowessCasketsPower are integers (effect:getPower())

Options:

1. I could change the casket droprate in settings to use 1-100 instead of a 0.0-1.0 float.


2 I could multiply the settings drop rate by 100 to convert it into an integer.

    `if math.random(1, 100) < utils.clamp((xi.settings.main.CASKET_DROP_RATE * 100) + kupowersMMBPower + prowessCasketsPower, 0, 100) then
        return true
    end`
    
Whichever style/option you prefer.

Reply:
If it's a much wider change, then leave it as it is 👍